pkg/core/native/native_test/management_test.go

```diff
@@ -205,6 +205,11 @@ func TestManagement_ContractDeploy(t *testing.T) {
 		t.Run("get after deploy", func(t *testing.T) {
 			managementInvoker.Invoke(t, si, "getContract", cs1.Hash.BytesBE())
 		})
+		t.Run("hasMethod after deploy", func(t *testing.T) {
+			managementInvoker.Invoke(t, stackitem.NewBool(true), "hasMethod", cs1.Hash.BytesBE(), "add", 2)
+			managementInvoker.Invoke(t, stackitem.NewBool(false), "hasMethod", cs1.Hash.BytesBE(), "add", 1)
+			managementInvoker.Invoke(t, stackitem.NewBool(false), "hasMethod", cs1.Hash.BytesLE(), "add", 2)
```

Comment:
Not sure, what's the difference between third and first cases, why it's `false` expected in the third case?

Reply:
Because of the wrong hash! BE/LE.

Reply:
Oh, right.